.github/workflows/iroha2-dev-code-quality.yml

```diff
@@ -3,10 +3,10 @@ name: I2::Dev::CodeQuality
 on:
   workflow_run:
     workflows: ["I2::Dev::Tests", "I2::Dev::Static"]
-    types: [requested]
+    types: [in_progress]
```

Comment:
May be it's possible even to set  `types: completed` to reduce parallels runs stronger? Btw, I think we want to start clippy and coverage jobs without waiting for finishing all other tests successfully.